### PR TITLE
PR: UI

### PR DESCRIPTION
**Pull request by pavan6115**
- Added all necessary pages
- Added components

[Preview](https://automophile-git-ui-kpavan.vercel.app/)

### CODE REVIEW

**Review by ayush65 — APPROVED** [review]
Code looks clean good to merge

File: src/pages/HomePage/HomePage.js [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+            <VCard thumbnail={spicsum} title={'Cars'} />
+            <VCard thumbnail={spicsum} title={'Cars'} />
+            <VCard thumbnail={spicsum} title={'Cars'} />
+            <VCard thumbnail={spicsum} title={'Cars'} />
+            <VCard thumbnail={spicsum} title={'Cars'} />
+            <VCard thumbnail={spicsum} title={'Cars'} />
+            <VCard thumbnail={spicsum} title={'Cars'} />
+            <VCard thumbnail={spicsum} title={'Cars'} />
+            <VCard thumbnail={spicsum} title={'Cars'} />
+          </div>
```
Comment (anjali1102) [inline src/pages/HomePage/HomePage.js:174]: we can do .map()

File: src/pages/MoviesPage/MoviePage.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+export const MoviePage = () => {
+  return (
+    <div>
+      <Navbar />
+      <div className='movie-container'>
+        <h2 className='movie-title'>Movies</h2>
+        <div className='movie-card-container'>
+          <HCard
+            thumbnail={moviepicsum}
```
Comment (anjali1102) [inline src/pages/MoviesPage/MoviePage.js:14]: thumbnail you don't need to  need to add your own image, 
can be done by inspecting youtube thumbail

Comment (anjali1102) [reply src/pages/MoviesPage/MoviePage.js:14]: also title={' '}  time={' '} this need not to be done manually, it can be taken from data.js

**Review by anjali1102 — APPROVED** [review]
liked you explored new things, useRef, clean coding by css variable, barrel export 
UI is beautiful ✨
looks good to merge. changes you can do later.

File: src/pages/HomePage/HomePage.js (hunk 1 above)
Comment (pavan6115) [reply src/pages/HomePage/HomePage.js:174]: yupp will do that, added more components just for demo purpose, once I update db, will pull everything from there
Thanks for the suggestion

File: src/pages/MoviesPage/MoviePage.js (hunk 2 above)
Comment (pavan6115) [reply src/pages/MoviesPage/MoviePage.js:14]: yes 👍

**Pull request merged**